### ISSUE TITLE
Regarding the evaluation of CLIP performance after fine-tuning

### ISSUE THREAD

**Issue by lianxxx** [uvavision/TV-GZSL#1]
Thanks for sharing the amazing work. I'd like to ask about a few details: 
(i) What do you think are critical things to ensure a successful finetuning of CLIP on these datasets with particularly specific class names, especially how could finetuning with seen classes improve the accuracy on unseen classes? Did you finetune the entire CLIP network or just a few layers? 
(ii) The results of Figure 5 show using attributes does not help improve the performance, which is somewhat unexpected. What are the possible reasons for that? How were the text prompts created with the attributes? It seems one of the attributes greater than a threshold was randomly selected to generate the text prompt. Was this way used for both finetuning and testing? 
https://github.com/uvavision/TV-GZSL/blob/3f6f5e5dcf2a8116920d9cc8459dac2eb13b9ced/finetune/dataloader.py#L113
https://github.com/uvavision/TV-GZSL/blob/3f6f5e5dcf2a8116920d9cc8459dac2eb13b9ced/finetune/dataloader.py#L136

Thank you very much!
